src/pages/NewResponses.vue

```diff
@@ -36,12 +65,59 @@ export default defineComponent({
       useTitle(ref(`${qdata.data.title}`))
     })
 
+    const keepNewResponses = () => {
+      const responses = makeResponses(true)
+      apis.postResponse(responses)
+    }
```

Comment:
```suggestion
    const keepNewResponses = async () => {
      const responses = makeResponses(true)
      await apis.postResponse(responses)
    }
```
非同期関数を使う関数は基本的に async で囲んだ方がいいです(関数の終了タイミングが明確なため)

Reply:
answerNewResponses も